client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -26,10 +26,13 @@ const InvestigationTable: React.FC = (): JSX.Element => {
 
     const [selectedRow, setSelectedRow] = React.useState<number>(UNDEFINED_ROW);
     const [investigator, setInvestigator] = React.useState<Investigator>(defaultInvestigator);
-
+    const [order, setOrder] = React.useState<Order>('asc');
```

Comment:
You should place the 'asc' and 'dsc' values on an enum

Reply:
done